### PR TITLE
Displaying non-selectable texts as pseudo elements

### PR DESCRIPTION
**Pull request by mshoho**
Displaying non-selectable texts as pseudo elements to avoid copying them to clipboard on web.

### CODE REVIEW

File: src/web/Text.tsx [hunk 1]
```diff
@@ -15,6 +15,14 @@ import RX = require('../common/Interfaces');
 import Styles from './Styles';
 import Types = require('../common/Types');
 
+if (typeof document !== 'undefined') {
```
Comment (erictraut) [inline src/web/Text.tsx]: Please add a comment that explains what this does. It's sufficiently non-obvious that a comment is needed. :)

File: src/web/Text.tsx [hunk 2]
```diff
@@ -15,6 +15,14 @@ import RX = require('../common/Interfaces');
...
+    const textAsPseudoElement = '[data-text-as-pseudo-element]::before { content: attr(data-text-as-pseudo-element); }';
+    const style = document.createElement('style');
```
Comment (erictraut) [inline src/web/Text.tsx]: This dynamic modification of the style is clever. I think we should use this same technique to eliminate the manual "box-sizing" override step that's currently needed for all ReactXP apps: https://microsoft.github.io/reactxp/docs/using-reactxp.html. If you feel like it, you're welcome to make this change as well -- and remove the step from the documentation.

Comment (mshoho) [inline src/web/Text.tsx]: Is RootView.tsx the best place to add it?

**Comment by erictraut** [comment]
Yeah, that makes sense.

File: src/web/Text.tsx (hunk 1 above)
Comment (mshoho) [inline src/web/Text.tsx]: Done.

**Pull request merged**